crates/holochain/src/core/ribosome/real_ribosome.rs

```diff
@@ -1152,13 +986,101 @@ pub mod wasm_test {
     use crate::core::ribosome::wasm_test::RibosomeTestFixture;
     use crate::core::ribosome::ZomeCall;
     use crate::sweettest::SweetConductor;
+    use crate::sweettest::SweetConductorConfig;
     use crate::sweettest::SweetDnaFile;
+    use crate::sweettest::SweetLocalRendezvous;
     use ::fixt::prelude::*;
     use hdk::prelude::*;
     use holochain_nonce::fresh_nonce;
     use holochain_types::prelude::AgentPubKeyFixturator;
     use holochain_wasm_test_utils::TestWasm;
     use holochain_zome_types::zome_io::ZomeCallUnsigned;
+    use std::sync::Arc;
+    use std::time::Duration;
+
+    #[tokio::test(flavor = "multi_thread")]
+    // guard to assure that response time to zome calls and concurrent zome calls
+    // is not increasing disproportionally
+    async fn concurrent_zome_call_response_time_guard() {
```

Comment:
this tests concurrent zome calls and their response times and is meant as an integration level guard to prevent unreasonable increases of zome call response time